css/components/modal/_modal-body.scss

```diff
@@ -0,0 +1,159 @@
+.modal-body {
+  display: inline-block;
+  height: auto;
+  overflow-y: scroll;
+  position: relative;
+  width: 100%;
+
+  .wp-editor-area {
+    resize: vertical;
+  }
+
+  .wp-editor-wrap,
+  .wp-editor-container,
+  .wp-editor-container
+  .wp-editor-area {
+    height: 100%;
+  }
+
+  >form {
```

Comment:
Sounds good; I'll add another directory for `form` and then create three files, `_form.scss` to contain the shared parts, and then `_form-mc.scss` and `_form-code.scss` where it swaps between `multiple-choice` and `coding` questions. :+1: